nixos/modules/services/misc/gitit.nix

```diff
@@ -665,9 +665,9 @@ in
       wantedBy = [ "multi-user.target" ];
       path = with pkgs; [ curl ]
              ++ optional cfg.pdfExport texlive.combined.scheme-basic
-	     ++ optional (cfg.repositoryType == "darcs") darcs
-	     ++ optional (cfg.repositoryType == "mercurial") mercurial
-	     ++ optional (cfg.repositoryType == "git") git;
+         ++ optional (cfg.repositoryType == "darcs") darcs
+         ++ optional (cfg.repositoryType == "mercurial") mercurial
+         ++ optional (cfg.repositoryType == "git") git;
```

Comment:
I got rid of the tabs myself in https://github.com/NixOS/nixpkgs/pull/136909/commits/e7b8d047791160047bc21b4b6bc6d29394ddbe6d because they were making editorconfig [complain](https://github.com/NixOS/nixpkgs/runs/3778846550). (I think this is because this file was last touched in 2018 but the editorconfig test is more recent)
I'll try and fix the indentation.

Reply:
There, needed 8 spaces instead of 4.